depends/common/libusb/CMakeLists.txt

```diff
@@ -6,10 +6,12 @@ include(ExternalProject)
 ExternalProject_Add(
     libusb
     SOURCE_DIR          ${CMAKE_SOURCE_DIR}
-    CONFIGURE_COMMAND   <SOURCE_DIR>/autogen.sh
-                            --prefix=${OUTPUT_DIR}
+    CONFIGURE_COMMAND   <SOURCE_DIR>/bootstrap.sh
+    COMMAND             <SOURCE_DIR>/configure
+                            --prefix=${CMAKE_INSTALL_PREFIX}
```

Comment:
Changing `prefix` from `${OUTPUT_DIR}` to `${CMAKE_INSTALL_PREFIX}` doesn't look right.